app/templates/base.html

```diff
@@ -5,10 +5,45 @@
 	<title>Wikipath</title>
 	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
 	<link rel="stylesheet" href="{{ url_for('static',filename='styles/main.css') }}">
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+	<script src="{{ url_for('static',filename='front-end-scripts/login.js') }}"></script>
 </head>
 
-<body>
-	{%block content%}{%endblock%}
+<body id="override">
+	<!-- Create the header of the page using Bootstrap -->
+	<nav class="navbar navbar-expand-md navbar-light bg-light">
+		<div class="container-fluid">
+			<a class="navbar-brand">WikiPath</a>
+			<button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+				data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+				aria-label="Toggle navigation">
+				<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse" id="navbarSupportedContent">
+				<ul class="nav nav-pills nav-fill me-auto mb-2 mb-lg-0">
+					<li class="nav-item">
+						<a class="nav-link active" href="#">Play</a>
+					</li>
+					<li class="nav-item">
+						<a class="nav-link" href="#">Create</a>
```

Comment:
href should be {{url_for(create)}} so we can actually navigate to the create page